stack.cabal

```diff
@@ -219,7 +219,7 @@ library
                    , vector-binary-instances
                    , yaml >= 0.8.10.1
                    , zlib >= 0.5.4.2 && < 0.7
-                   , deepseq >= 1.4
+                   , deepseq >= 1.3 && < 1.5
```

Comment:
I agree that this is a somewhat ugly situation but as long as it compiles it seems to be safe enough.

This [stackoverflow answer](http://stackoverflow.com/questions/9301122/haskell-cabal-package-indirectly-depends-on-multiple-versions-of-the-same-pack) explains it rather well.